src/App.css

```diff
@@ -1,38 +1,38 @@
-.App {
-  text-align: center;
-}
+/*.App {*/
```

Comment:
It doesn't seem like this file is being used anymore, worth deleting it and removing it's import from App.js